pkg/cmd/repo/license/view/view.go

```diff
@@ -0,0 +1,99 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	License    string
+}
+
+func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Command {
+	opts := &ViewOptions{
+		IO:         f.IOStreams,
+		HTTPClient: f.HttpClient,
+		Config:     f.Config,
+		License:    "",
+	}
+
+	cmd := &cobra.Command{
+		Use:   "view <license>",
+		Short: "View an available repository license template",
+		Long: heredoc.Docf(`
+		View an available repository license template.
+		
+		%[1]s<license>%[1]s is a license name or SPDX ID.
+		`, "`"),
+		Args: cmdutil.ExactArgs(1, "gh repo licese view only takes a single license argument"),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.License = args[0]
+			if runF != nil {
+				return runF(opts)
+			}
+			return viewRun(opts)
+		},
+	}
+	return cmd
+}
+
+func viewRun(opts *ViewOptions) error {
+	if opts.License == "" {
+		return errors.New("no license provided")
+	}
+
+	client, err := opts.HTTPClient()
+	if err != nil {
+		return err
+	}
+
+	cfg, err := opts.Config()
+	if err != nil {
+		return err
+	}
+
+	if err := opts.IO.StartPager(); err == nil {
+		defer opts.IO.StopPager()
+	} else {
+		fmt.Fprintf(opts.IO.ErrOut, "failed to start pager: %v\n", err)
+	}
+
+	hostname, _ := cfg.Authentication().DefaultHost()
+	license, err := api.LicenseTemplate(client, hostname, opts.License)
+	if err != nil {
+		if strings.Contains(err.Error(), "HTTP 404") {
+			return fmt.Errorf("'%s' is not a valid license template name or SPDX ID. Run `gh repo license list` for options", opts.License)
+		}
+		return err
+	}
+
+	return renderLicense(license, opts)
+}
+
+func renderLicense(licenseTemplate api.License, opts *ViewOptions) error {
+	cs := opts.IO.ColorScheme()
+	var out strings.Builder
+	if opts.IO.IsStdoutTTY() {
+		out.WriteString(fmt.Sprintf("\n%s\n", cs.Gray(licenseTemplate.Description)))
+		out.WriteString(fmt.Sprintf("\n%s\n", cs.Grayf("To implement: %s", licenseTemplate.Implementation)))
+		out.WriteString(fmt.Sprintf("\n%s\n\n", cs.Grayf("For more information, see: %s", licenseTemplate.HTMLURL)))
+	}
+	out.WriteString(licenseTemplate.Body)
+	_, err := opts.IO.Out.Write([]byte(out.String()))
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
1. **Should users be able to redirect stdout to a license file to contain just the license body?**

   If so, then we would need to use stderr for informational.
1. **Do we use other colors when communicate actions for users to take?**

   Not sure dim / gray is the right choice to direct the user for taking action.

Reply:
> Should users be able to redirect stdout to a license file to contain just the license body?

I think it already does this.

```shell
❯ ./bin/gh repo license view mit > license.txt

❯ cat license.txt 
MIT License

Copyright (c) [year] [fullname]

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.

```

> Do we use other colors when communicate actions for users to take?

Open to suggestions 😁 I just wanted it to be distinct from the actual license.

Reply:
Ooh, I get it. 🤔 Well done.

Reply:
I suppose I was thinking about how we use yellow [here](https://github.com/cli/cli/blob/76ea9396275ac5a79d62f6987303b7e12b147ff9/cmd/gh/main.go#L174-L191) for calling attention to `gh` upgrade, which aligns with the emphasis [Primer color role](https://primer.style/foundations/color/overview#color-roles).

That said, it isn't a deal breaker if a specific color; maybe not dimming it?  I'm open to ideas.